actors/miner/src/testing.rs

```diff
@@ -542,8 +542,10 @@ impl PartitionStateSummary {
 
                 expiration_epochs = queue_summary.expiration_epochs;
                 // check the queue is compatible with partition fields
-                let queue_sectors =
-                    BitField::union([&queue_summary.on_time_sectors, &queue_summary.early_sectors]);
+                let queue_sectors = BitField::union([
+                    &queue_summary.on_time_sectors,
+                    &queue_summary.faulty_sectors,
```

Comment:
And early sectors

Reply:
I'm working on adding tests for parition.